app/assets/stylesheets/components/_flexbox.scss

```diff
@@ -0,0 +1,34 @@
+.row {
```

Comment:
Could have used directly bootstrap grid system or manual grid instead of flex for this section

Reply:
Also, careful when naming things exactly as bootstrap, as it may lead to weird and unexpected results in pages.

Reply:
_flebox has been deleted from the new branch... you can ignore it

Reply:
ahh, right, thanks for reminding!